cmake/CMakeLists.txt

```diff
@@ -426,6 +426,32 @@ if(PKG_KIM)
   include_directories(${KIM_INCLUDE_DIRS})
 endif()
 
+if(PKG_MESSAGE)
+  option(MESSAGE_ZMQ "Use ZeroMQ in MESSAGE package" OFF)
+  file(GLOB_RECURSE cslib_SOURCES ${LAMMPS_LIB_SOURCE_DIR}/message/cslib/*.F
```

Comment:
Could we move that glob part in the `SIMPLELIB` loop?

Reply:
have a look at the lib and the source folder, the names don't match. that's why I did it this way.

Reply:
Good point!